src/app/user/profile/mobile/components/MobileProfileSection/ResearcherProfileTab/MobileMyPosts/AllMenuBottomSheet/AllMenuBottomSheet.tsx

```
@@ -0,0 +1,146 @@
+  postId: string;
+  initialRecruitStatus: boolean;
+  currentPage: number;
+  onRecruitComplete?: { onSuccess?: () => void; onError?: () => void };
+  onDelete?: { onSuccess?: () => void; onError?: () => void };
+}
+
+const AllMenuBottomSheet = ({
+  onClose,
+  postId,
+  initialRecruitStatus,
+  currentPage,
```

ESLint

[error] 'currentPage' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)